Streetcode/Streetcode.BLL/MediatR/Streetcode/Streetcode/Create/CreateStreetcodeHandler.cs

```diff
@@ -343,7 +346,7 @@ private async Task AddTimelineItems(StreetcodeContent streetcode, IEnumerable<Ti
                .Select(x => new HistoricalContextTimeline
                {
                    HistoricalContextId = x.Id == 0
-                       ? newContextsDb.FirstOrDefault(h => h.Title.Equals(x.Title)).Id
+                       ? newContextsDb.FirstOrDefault(h => h.Title!.Equals(x.Title)) !.Id
```

Comment:
if you know that the result won't be `null` just use `First `instead of `FirstOrDefault`